examples/next-prisma-starter/src/pages/index.tsx

```diff
@@ -37,9 +37,8 @@ const IndexPage: NextPageWithLayout = () => {
     <>
       <h1>Welcome to your tRPC starter!</h1>
       <p>
-        Check <a href="https://trpc.io/docs">the docs</a> whenever you get
-        stuck, or ping <a href="https://twitter.com/alexdotjs">@alexdotjs</a> on
-        Twitter.
+        Check <a href="https://trpc.io/">the docs</a> whenever you get stuck, or
```

Comment:
i pointed it to the landingpage but we could also point it to /docs/v10 if you prefer that.